.github/workflows/main.yml

```diff
@@ -12,17 +12,18 @@ jobs:
     name: Test Code
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.8
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        python -m pip install git+https://github.com/mathurinm/celer.git
-        pip install pytest
-        pip install numpydoc
-        pip install .
-    - name: Test with pytest
-      run: pytest -v skglm/
+      - uses: actions/checkout@v2
```

Comment:
@PABannier can you revert this big chunk of changes ? Normally there's just one line that changed